src/som/vmobjects/SClass.java

```diff
@@ -302,4 +307,18 @@ public String toString() {
 
   // Static field indices and number of class fields
   static final int numberOfClassFields = numberOfObjectFields;
+
+  /**
+   * INTEROP
+   * A cast of a SClass to a java String will return the name of the class 
+   */
+  @ExportMessage
+  public boolean isString() {
+    return true;
```

Comment:
Hm, this feels wrong. We can't simply do the same as TruffleSOM, and get the Java object out directly?

Reply:
Indeed, I went back to the TruffleSOM way of handling the java object